argoverse/utils/sim2.py

```diff
@@ -108,3 +109,22 @@ def transform_from(self, point_cloud: np.ndarray) -> np.ndarray:
     def transform_point_cloud(self, point_cloud: np.ndarray) -> np.ndarray:
         """Alias for `transform_from()`, for synchrony w/ API provided by SE(2) and SE(3) classes."""
         return self.transform_from(point_cloud)
+
+    @classmethod
+    def from_json(cls, json_fpath: str) -> "Sim2":
```

Comment:
What do you think about a `Union[_Pathlike, str]` here?

Reply:
At some point we should define `Union[_Pathlike, str]` in a `typing` module instead of copy-pasting throughout the library IMO.

Reply:
Good point about the PathLike -- I just updated it